### PR TITLE
Asks user for a suitable image when one is not available on the Web Manifest

### LINKED ISSUE #12: Building fails if the icon is less than 512x512
If you run the tool on a webapp that doesn't have an icon larger than 512x512, it won't work.

I don't think icons of that size are currently required for anything else, and I've found two major webapps that don't have such an icon (Pinterest and Instagram).

It would be good if we could either remove the need or let the user provide another image during `llama-pack init` instead of having to change their webapp manifest.

### PR DESCRIPTION
**Pull request by andreban**
Closes #12

### CODE REVIEW

**Comment by andreban** [comment]
@NotWoods I think this affects your ongoing work on Adaptive Icons, but seems like a move in the right direction.

**Comment by NotWoods** [comment]
I can probably add in an optional prompt for the maskable icon, following the same format.

**Comment by andreban** [comment]
Sounds good to me.

**Comment by andreban** [comment]
@NotWoods rebased with master and added a prompt for the maskable icon. I tested and things seems to work as intended. Do you mind taking a quick look if the maskable bits look right?

File: cli/cmds/init.js [hunk 1]
```diff
@@ -77,6 +78,21 @@ async function createTwaConfig(manifestUrl, manifest) {
...
+        name: 'iconUrl',
+        description: 'URL to an image that is at least 512x512px',
+        message: 'Must be a well-formed http or https URL.',
+        required: true,
+        default: icon ? new URL(icon.src, manifestUrl).toString() : '',
+        conform: validUrl.isWebUri,
+      },
+      maskableIconUrl: {
+        name: 'maskableIconUrl',
+        description: 'URL to an image to be used when generating maskable icons',
```
Comment (NotWoods) [inline cli/cmds/init.js]: It might be worth mentioning that this icon also has to be at least 512x512px

File: spec/lib/utilSpec.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          'type': 'image/png',
+        }],
+      }, 'any');
+      expect(result.src).toBe('/favicons/android-chrome-512x512.png');
+      expect(result.sizes).toBe('512x512');
+      expect(result.type).toBe('image/png');
+      expect(result.size).toBe(512);
+    });
+
+    it('returns null when not maskable icon is available', () => {
```
Comment (NotWoods) [inline spec/lib/utilSpec.js]: nit: 
```suggestion
    it('returns null when no maskable icon is available', () => {
```

**Pull request merged**